go.mod

```diff
@@ -5,4 +5,5 @@ go 1.14
 require (
 	github.com/google/gofuzz v1.2.0
 	github.com/lazyledger/merkletree v0.0.0-20201214195110-6901c4c3c75f
+	github.com/lazyledger/rsmt2d v0.0.0-20210121155026-23bf69f65287
```

Comment:
IMO, the NMT should not need to know about rsmt2d. That is why I think the wrapper should live in ll-core. There all the dependencies need to come together anyways. And introducing the dependency here might cause some headaches in the future.

Reply:
So what's the plan here, is the logic in this PR going to be moved instead of lazyledger-core?